highfidelity/hifi
# WL 21446: Add doppleganger "move-as-you-move" roomscale functionality

#### Testing (Roomscale HMD required):

* Launch this PR's marketplace script:
  - https://raw.githubusercontent.com/humbletim/hifi/b5f5a037/unpublishedScripts/marketplace/doppleganger-attachments/dist/app-doppleganger-marketplace.js
* Activate the doppleganger mirror.
* Using *hand controllers*: move around and examine the doppleganger.
  - Verify that it "stays put as you move."
* Using *your legs and body*: move around (within roomscale) and examine the doppleganger.
  - Verify that it "moves as you move."
   (ie: that it mimics your horizontal steps like forward/backward and sideways).
* Perform a quick doppleganger smoke test by adding/removing/observing attachments to verify overall behavior has remained consistent.
